pkg/sql/tests/sysbench_test.go

```diff
@@ -459,36 +472,26 @@ func (s *sysbenchKV) indexKey(t tableNum, k kValue, id rowID) []byte {
 	return key
 }
 
-func (s *sysbenchKV) pkValue(rng *rand.Rand, pkKey roachpb.Key, k kValue) []byte {
+func (s *sysbenchKV) pkValue(rng *rand.Rand, k kValue) []byte {
 	const pkValueSize = 185 // measured from sysbenchSQL
 	b := randutil.RandBytes(rng, pkValueSize)
 	s.encodePKValue(b, k)
-	var v roachpb.Value
-	v.SetBytes(b)
-	v.InitChecksum(pkKey)
-	return v.RawBytes
+	return b
 }
 
 func (s *sysbenchKV) encodePKValue(b []byte, k kValue) {
 	// Replace the first 8 bytes with the k value, so that we can decode k to keep
```

Comment:
The comment is stale.

Reply:
Actually it's not, nevermind.
<!-- Reviewable comment -ODu1iyt4oqJrnB2QwzY:b-saj7te -->
<!-- Sent from Reviewable.io -->